tests/Unit/ProxyClient/HttpDispatcherTest.php

```diff
@@ -171,6 +171,8 @@ public function testBanWithoutBaseUri()
 
         $request = $this->messageFactory->createRequest('BAN', '/', ['X-Url' => '/foo/.*']);
         $httpDispatcher->invalidate($request, false);
+
+        // TODO: This is already a risky case because it doesn't test anything. Should be fixed in master?
```

Comment:
@dbu I added https://github.com/FriendsOfSymfony/FOSHttpCache/issues/447 and will update this PR and remove this TODO because it's not related to this PR at all.